src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
```

Comment:
The attributes for the `html` tag should be indented with 2 spaces to maintain consistency with the rest of the document.